### PR TITLE
refactor(website): shadow on showcase toggle

### PR DESCRIPTION
**Pull request by dsmmcken**
## Motivation

Shadow belongs on the label, so that it gets rounded, and fix inner radius aliasing issue by overlapping. Use ::after instead of unnecessary element.

before:
142441432-5e750ce2-bdc9-4d60-b4d9-8743a27c80b2

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

After:
image

## Related PRs

#5742

### CODE REVIEW

**Comment by slorber** [comment]
Hey

I was working on a showcase PR too here:

https://github.com/facebook/docusaurus/pull/5969

I'd rather merge mine first, eventually apply your changes later ok?

**Comment by slorber** [comment]
PR is merged, if you want to backport your changes? comp is moved at website/src/pages/showcase/_components/ShowcaseFilterToggle

**Review by Josh-Cena — APPROVED** [review]
LGTM! This removed a lot of magic numbers in the CSS and used Infima variables, nice 👍

**Comment by dsmmcken** [comment]
Merged it to early, needed to remove `shadow--md` class from the component after @slorber refactor.

**Comment by Josh-Cena** [comment]
@dsmmcken Isn't it already removed?

File: website/src/pages/showcase/_components/ShowcaseFilterToggle/index.tsx [hunk 1]
```diff
@@ -55,7 +55,6 @@ export default function ShowcaseFilterToggle(): JSX.Element {
       <label htmlFor={id} className={clsx(styles.checkboxLabel, 'shadow--md')}>
```
Comment (slorber) [inline website/src/pages/showcase/_components/ShowcaseFilterToggle/index.tsx:55]: I added `shadow--md` here but it looks like it's also in the `. checkboxLabel` class so it's duplicated

**Pull request merged**